gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.internal.http.connectionTimeout=60000
 org.gradle.internal.http.socketTimeout=60000
 
-version=0.11.3
+version=0.11.4-SNAPSHOT
```

Comment:
_Minor:_ Your properties file might be out of sync? (`0.11.4-SNAPSHOT` is the latest).